### PR TITLE
add min, max, range, distances

### PR DESCRIPTION
**Pull request by SotirisGP**
added min, min distance from 0, max, max distance from 0 and range on a list of numbers

### CODE REVIEW

File: math/find_min_max.go [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+// Find the range.
+
+package main
+
+import (
+	"fmt"
+)
+
+// absolute value, also implemented in "math" as math.Abs
+func abs(a float64) float64 {
```
Comment (nomisrevol) [inline math/find_min_max.go:12]: Please reuse the existence function.

File: math/find_min_max.go [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+		}
+	}
+	return max
+}
+
+func findRange(numbers []float64) float64 {
+	return max(numbers) - min(numbers)
+}
+
+func main() {
```
Comment (nomisrevol) [inline math/find_min_max.go:67]: Remove the `main` function and write the test file instead.

File: math/find_min_max.go [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+	for _, n := range numbers {
+		if n > max {
+			max = n
+		}
+	}
+	return max
+}
+
+//find the largest distance from 0
+func maxDist(numbers []float64) float64 {
```
Comment (nomisrevol) [inline math/find_min_max.go:53]: Should rename to `maxDistToZero`, I think it is more meaningful.

**Review by nomisrevol — CHANGES_REQUESTED** [review]
Overall, you are naming things not really good, trying to see this doc as reference: https://github.com/unknwon/go-code-convention/blob/master/en-US/naming_rules.md#:~:text=Variable%20name%20is%20generally%20using,e.g.%20APIClient%20%2C%20repoID%20%2C%20UserID%20.

Also, find a more meaningful variable name for your variables.

**Comment by tjgurwara99** [comment]
Please take a look at our new [contribution guidelines](https://github.com/TheAlgorithms/Go/blob/d2f12849df76203d80762ceca5105531d3424384/CONTRIBUTING.md) and make changes accordingly. Currently this PR is not up to standard. If no changes are made in the next two weeks, I will consider this PR to be stale and will close this PR then.

**Pull request closed without merging**